registry/extension/oras/referrers.go

```diff
@@ -53,3 +116,13 @@ func (h *referrersHandler) getReferrers(w http.ResponseWriter, r *http.Request)
 		return
 	}
 }
+
+func generateLinkHeader(repoName, subjectDigest, artifactType string, lastDigests []string, nPage int) string {
+	url := fmt.Sprintf("/v2/%s/_oras/artifacts/referrers?digest=%s&artifactType=%s&n=%d&nextToken=%s",
```

Comment:
nit: if `artifactType` is not provided in the original request, it's better to not add it to the next link, Same for other query params.

Reply:
This is true (As a note I included this in my version of this method in our internal implementation for reference)

Reply:
removed artifactType and n from link header if not provided in original request